src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -32,6 +39,8 @@ interface Props {
 interface State {
   isPopoverOpen: boolean;
   curlCode: string;
+  toasts: [];
```

Comment:
Please remove all uses of `// @ts-ignore`. You will probably have to add some type information here :)

Reply:
Ok, done.